src/plugins/jni/jni.c

```diff
@@ -38,6 +38,8 @@ typedef struct
 	int printException;
 	jmethodID midKeyConstr;
 	jmethodID midKeySetConstr;
+	// #3825 comment out the following 1 line if reference counter increase on Java Key creation is dissabled
```

Comment:
please make TODO marker at such places

```suggestion
	// TODO #3825 comment out the following 1 line if reference counter increase on Java Key creation is dissabled
```

Reply:
will do, hopefully I've got all places to be updated